packages/dashboard/src/components/feature-graph/index.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect } from 'react';
+import { cluster, line, curveBundle, select, hierarchy, HierarchyPointNode } from 'd3';
+import { classes } from './styles.st.css';
+
+export interface Link {
+    source: string;
+    target: string;
+}
+
+interface Node {
+    name: string;
+    id: string;
+    group: number;
+    parent?: GraphNode;
+}
+
+export interface GraphData {
+    nodes: Node[];
+    links: Link[];
+}
+
+export interface IFeatureGraphProps {
+    selectedFeatureGraph: GraphData;
+}
+
+interface GraphNode {
+    id?: string;
+    name: string;
+    group: number;
+    children: Array<GraphNode | Node>;
+    parent?: GraphNode;
+}
+
+export const FeatureGraph = ({ selectedFeatureGraph }: IFeatureGraphProps) => {
+    useEffect(() => {
+        const diameter = 600;
+        const radius = diameter / 2;
+        const innerRadius = radius - 70;
+
+        const graphCluster = cluster<GraphNode>()
+            .size([360, innerRadius])
+            .separation(function (a, b) {
```

Comment:
1? 2? 4? 
please add a comment explaining this numbers

Reply:
Removed the function, it's actually irreelevant